### ISSUE TITLE
Flow diagrams are difficult to navigate

### ISSUE THREAD

**Issue by ColinDaglish** [datasciencecampus/coding-in-the-open#1]
**Describe the bug**
The flow diagram on the private to public page is quite chunky, meaning the user has to scroll down to be able to scroll left to right, in order to see anything. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to ['private_to_public'](https://datasciencecampus.github.io/coding-in-the-open/private_to_public/guidance.html)
2. Try to see what is in box 4 without needing to scroll

**Expected behaviour**
Ideally this information should be condensed down so it can be viewed on one page without the need for scrolling. 

**Screenshots**
![image](https://github.com/datasciencecampus/coding-in-the-open/assets/87810570/cf603335-58f4-4368-9acd-9014ac0c1624)

**Environment (please complete the following information):**
Google Chrome

**Additional context**

**Comment by r-leyshon** [comment]
Agreed - known issue this one. We have a model we want to implement and I hope to turn that around prior to sharing next week.

**Comment by r-leyshon** [comment]
Fixed by #5
